homeassistant/components/flux_led/light.py

```diff
@@ -100,8 +111,11 @@
 
 FLUX_EFFECT_LIST = sorted(list(EFFECT_MAP)) + [EFFECT_RANDOM]
 
+SERVICE_CUSTOM_EFFECT = "set_custom_effect"
```

Comment:
Please don't add any new features in this PR, besides the config flow and config entries support. Try to minimize the PR.

Reply:
I can remove and move to another PR - was trying to keep functionality whole. Will base additional PR on this one.

Reply:
Just looking at it, the custom service is extremely small here in relation to the rest of the fixes/refactoring. Wondering if for the sake of the 10-15 lines of code and pretty isolated, can we just keep it in here so the functionality from the original integration doesn't get deprecated (custom effects) as a result of this update.